pyonfx/shape.py

```diff
@@ -17,8 +17,10 @@
 
 import re
 import math
+from typing import Callable
 from pyquaternion import Quaternion
 from inspect import signature
+from __future__ import annotations
```

Comment:
Yes it is necessary because

```python
def __eq__(self, other: Shape):
```

`other` has the same type with `self`, enclosed by its own class (`Shape`). Removing it will cause Pylance to yield

```
"Shape" is not defined
```

Reply:
Thank you, I didn't know about that.